.github/workflows/node.js.yml

```diff
@@ -2,18 +2,16 @@ name: CI/CD
 
 on:
   push:
-    branches: [ master ]
-  pull_request:
-    branches: [ master ]
-
+    branches: [ ch-setup-continous-continous-integration-#18 ]
```

Comment:
Can you add more branches? Preferable your branch and develop branch. So as to allow them to trigger the workflow run.